dexcom/egv.go

```diff
@@ -185,7 +185,7 @@ func (e *EGV) Parse(parser structure.ObjectParser) {
 	e.ID = parser.String("recordId")
 	e.SystemTime = TimeFromString(parser.String("systemTime"))
 	e.DisplayTime = TimeFromString(parser.String("displayTime"))
-	e.Unit = parser.String("unit")
+	e.Unit = UnitFromParser(parser, EGVUnitMgdL)
 	e.RateUnit = parser.String("rateUnit")
```

Comment:
Likely need same default handling for `rateUnit` as well

Reply:
Also for the `unit` in calibrations (https://github.com/tidepool-org/platform/blob/master/dexcom/calibration.go)

Reply:
I agree with `rateUnit` but question the unit in `calibration`.

Looking at both our code and the dexcom API it is allowed for the units to potentially not be set 

- https://github.com/tidepool-org/platform/blob/cd87510bb586c45a52c69f11680cba48a357ff6a/dexcom/calibration.go#L144-L153
- https://developer.dexcom.com/docs/dexcomv3/operation/getCalibrationsV3/#!c=200&path=records/unit&t=response

Reply:
Sorry, I should have been more clear. I did notice that the calibration record will pass validation as-is.

In the other cases (EGVs, events) though, the proposed solution has the parser filling in the missing unit with a default value, thereby passing the subsequent validation. That also means that the subsequent DB record will end up with that filled-in (default) unit.